grabbit/src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -137,11 +138,27 @@ class GrabbitConfiguration {
 
     @CompileStatic
     static class PathConfiguration {
+        private static final String COMMA = ","
+        private static final String SLASH = "/"
+        private static final String EMPTY = ""
         String path
+        Collection<String> excludePaths
         Collection<String> workflowConfigIds
+        
+        Collection<String> modifyExcludePaths(@Nonnull Collection<String> excludePaths)
+        {
+            Collection<String> temp = excludePaths
+            String modifiedExcludePaths = EMPTY;
+            for(String append : temp)
+            {
+                modifiedExcludePaths += this.path+SLASH+append+COMMA;
+            }
+            return modifiedExcludePaths.split(COMMA) as Collection<String>
```

Comment:
To be very specific about watching out for assumptions about the contents of an exclude path, see http://www.ietf.org/rfc/rfc2396.txt which talks about what's legal in a URI path. (Specifically section 2.2)